BraintreeApi/src/androidTest/java/com/braintreepayments/api/CoinbaseTest.java

```diff
@@ -0,0 +1,171 @@
+package com.braintreepayments.api;
+
+import android.content.Context;
+import android.content.Intent;
+import android.net.Uri;
+
+import com.braintreepayments.api.exceptions.AppSwitchNotAvailableException;
+import com.braintreepayments.api.exceptions.CoinbaseException;
+import com.braintreepayments.api.exceptions.ConfigurationException;
+import com.braintreepayments.api.models.CoinbaseConfiguration;
+import com.braintreepayments.api.models.Configuration;
+
+import org.junit.Test;
+
+import java.io.UnsupportedEncodingException;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class CoinbaseTest {
+
+    @Test
+    public void testIsAvailableReturnsTrueWhenCoinbaseIsEnabled() {
+        Configuration configuration = mock(Configuration.class);
+        when(configuration.isCoinbaseEnabled()).thenReturn(true);
+
+        Coinbase coinbase = getCoinbaseWithSpecifiedConfiguration(configuration);
```

Comment:
I feel like this method name is somewhat non-idiomatic, maybe something like: `getCoinbaseForConfiguration()` might be nicer.